frontend/src/scenes/web-analytics/pageReportsLogic.ts

```diff
@@ -117,7 +117,7 @@ FROM (
     SELECT
         e.session.session_id as session_id,
         min(e.timestamp) as ts,
-        avg(toFloat(e.properties.$prev_pageview_duration)) as session_avg_duration
+        avg(least(toFloat(e.properties.$prev_pageview_duration), 1800) as session_avg_duration
```

Comment:
i guess my fly-by question (and i'm very removed from this part of the product)

how many customers currently get more than 30 minutes as the result here? and how many of them _want_ more than 30 minutes 

this would confuse them, no?

if i have a video editing app then i could easily have hours on one page where the user is active

if I have an ecommerce app then 30 minutes would mean a user got stuck